src/mm/buddy/multi.h

```diff
@@ -40,4 +40,3 @@ struct mm_state {
 	/// The total count of allocated bytes
 	uint_fast32_t full_ckpt_size;
 };
-
```

Comment:
You really hate the trailing new line of this file!

Reply:
There's a wrong encoding there. I fixed it in some other commit that will be coming.